README.md

```diff
@@ -1,29 +1,111 @@
-:duck: Malduck
-=========
+# :duck: Malduck
 
 Malduck is your ducky companion in malware analysis journeys. It is mostly based on [Roach](https://github.com/hatching/roach) project, which derives many concepts from [mlib](https://github.com/mak/mlib) 
 library created by [Maciej Kotowicz](https://lokalhost.pl). The purpose of fork was to make Roach independent from [Cuckoo Sandbox](https://cuckoosandbox.org/) project, but still supporting its internal `procmem` format.
 
 Malduck provides many improvements resulting from CERT.pl codebase, making scripts written for malware analysis purposes much shorter and more powerful. 
 
-Improvements
-============
+## Features
 
-* Support for (non)memory-mapped PE images without header fix-up.
-* Searching for wildcarded byte sequences
-* Support for x64 disassembly
-* Fixed-precision integer types
-* Many improvements in ProcessMemory
+- **Cryptography** (AES, Blowfish, Camelie, ChaCha20, Serpent and many others)
+- **Compression algorithms** (aplib, gzip, inznt1 (RtlDecompressBuffer)
+- **Memory model objects** (work on memory dumps, PE/ELF, raw files and IDA dumps with the same code)
+- **Extraction engine** (modular extraction framework for config extraction from files/dumps)
+- Fixed integer types (like Uint64) and bitwise utilities
+- String operations (chunks, padding, packing/unpacking etc)
+- Hashing algorithms (CRC32, MD5, SHA1, SHA256)
 
-Usage
-==========
+## Usage examples
 
-Installing may be performed by running
+#### AES
+
+```python
+from malduck import aes
+
+key = b'A'*16
+iv = b'B'*16
+plaintext = b'data'*16
+ciphertext = aes.cbc.encrypt(key, iv, plaintext)
+```
+
+### Serpent
+
+```python
+from malduck import serpent
+
+key = b'a'*16
+iv = b'b'*16
+plaintext = b'data'*16
+ciphertext = serpent.cbc.encrypt(key, plaintext, iv=iv)
```

Comment:
It seems kinda weird to use kwargs here and nowhere else.

Reply:
I've actually mostly copied the examples from the docs. Do you think that it's a good idea to back-port your suggestions to the code examples in the main docs?

Reply:
Other weird thing: cbc.encrypt is (key, iv, data), and serpent.encrypt is (key, data, iv) :thinking:

Reply:
Hmh, doesn't seem good. I don't have any ideas how to fix it without breaking compatiblity though.